src/memtable.h

```diff
@@ -282,15 +282,15 @@ memtable_verify(cache *cc, memtable *mt)
 }
 
 static inline void
-memtable_print(cache *cc, memtable *mt)
+memtable_print(cache *cc, memtable *mt, platform_log_handle *log_handle)
```

Comment:
To reiterate ... I am seeing more print methods' interfaces now changing to take the `platform_log_handle *log_handle` as a new arg. Here, it's the 3rd arg.

I would have gone the other way (as suggested previously) of making this output `log_handle` consistently the 1st-arg in all print-methods.